data/flow/flowdata.py

```diff
@@ -33,6 +34,8 @@ def __init__(self, directory, batch_size=1, validation_size=1, crop_size=None, t
         :param training_augmentations: Whether to do live augmentations while training.
         :param data_source: Source of the data.
         :param augmentation_config: Configurations for data augmentation. If None, the default will be used.
+        :param max_flow: Float. Maximum flow magnitude of the flow image. Any examples with flow magnitude greater than
```

Comment:
What is this needed for?

Reply:
It said in the paper to discard examples with flow greater than 1000. In my debugging print statements, there are actually flyingthings examples with flows larger than this.